pages/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import Link from 'next/link'
 
-export default () => (
+const Home = () => (
```

Comment:
Hi, I tried running the demo, it seems that adding a hook stopped prefresh from working. I tested it by adding a `useState(0)` here. 

I was tinkering with prefresh on my own nextjs project and ran into a similar problem and I can't seem to figure out why 😅.  Does it have something to do with `react-refresh/babel`? 

I love that a nextjs plugin is coming out so fast btw, thx for working on it.

Reply:
It does seem to work correctly for me, what version of next are you using? If it's pre 9.4 this won't work unless you add the babel plugin yourrself

Reply:
Ah, apparently it's not caused by `@prefresh/nextjs`. It seems to be caused by preact-devtools on my end. I just tried it with preact-devtools plugin turned off and it runs perfectly. If I commented out the devtools injection code on `next.config.js`, it would also work. I'm using 9.4.

Reply:
Hmm odd that, going to try and figure out what's going wrong

Reply:
I wonder if a hook is being injected but the original isn't being called?